kuma/javascript/src/payments/subscription-form.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+import * as React from 'react';
+import { useRef, useState } from 'react';
+
+import { getLocale, gettext, Interpolated } from '../l10n.js';
+
+export default function SubscriptionForm() {
+    const locale = getLocale();
+    const subscriptionFormRef = useRef(null);
+    const [subscribeFormDisabled, setSubscribeFormDisabled] = useState(false);
+    const [subscribeButtonEnabled, setSubscribeButtonEnabled] = useState(false);
+
+    const toggleButton = () => {
+        setSubscribeButtonEnabled(!subscribeButtonEnabled);
+    };
+
+    /**
+     * Opens Stripe modal allowing a user to complete their subscription.
+     * @param {Object} event - The form submit event
+     */
+    const submit = event => {
+        event.preventDefault();
+
+        const subscriptionForm = subscriptionFormRef.current;
+        if (subscriptionForm) {
+            const formData = new FormData(subscriptionForm);
+            setSubscribeFormDisabled(true);
+
+            const stripeHandler = window.StripeCheckout.configure({
```

Comment:
This assumes that the stripe SDK .js has fully loaded. Is that safe?

Reply:
I do not know whether I would call it safe, but the user will have to have noticed the component, checked the terms checkbox and then click `Continue` :)

With a slow connection, that might not be enough time though.

What do you see as our options? Do we disable the form and perhaps do some kind of polling until `window.StripCheckout` is not undefined? I guess if we do that, we should present some message to the user, or show a spinner or else they will just think the form is broken.